packages/utils/upgrade/packup.config.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Can this be imported from the package...?
+ */
+import { defineConfig } from '@strapi/pack-up';
+
+export default defineConfig({
+  bundles: [
+    {
+      source: './src/cli/index.ts',
+      require: './dist/cli.js',
+    },
+  ],
+  externals: [], // not sure if we should update this?
```

Comment:
it automatically detects builtin modules & uses your deps from the package.json – you can _probably_ remove it?

Reply:
Thank you :slightly_smiling_face: